src/views/Home/index.js

```diff
@@ -1,9 +1,49 @@
 import React from 'react';
-import { Heading } from '../../shared/styles/HomeStyles'
+import { Button } from '@material-ui/core';
+import image1 from '../../assets/images/land-img-1.png';
+import image2 from '../../assets/images/land-img-2.png';
+import image3 from '../../assets/images/land-img-3.png';
+import image4 from '../../assets/images/land-img-4.png';
+import {
+  ContainerFluid, Header, BodyContainer, HighrightWrapper, HeadingWrapper,
+  ButtonsWrapper, ImagesWrapper, SubImagesWrapper
+} from '../../shared/styles/HomeStyles';
 
 const Home = () => {
-    return ( 
-        <Heading>Welcome to Barefoot Nomad with styled component</Heading>
+     return ( 
+        
+        <ContainerFluid>
+            {/* <Header></Header> */}
+            <BodyContainer>
+                <HighrightWrapper>
+                  <HeadingWrapper>
+                  <h1>TRAVEL BOOKINGS</h1> <h1>MADE EASY</h1>
+                    <ul>
+                      <li>Confort</li>
+                      <li>Food</li>
+                      <li>Luxury</li>
+                     </ul>
+                   </HeadingWrapper>
+                  <ButtonsWrapper>
+                  <Button variant="contained" color="primary" data-testid="viewHotels-btn">
+                    View Hotels
+                  </Button>
+                  <Button variant="contained" color="primary" data-testid="contactUs-btn">
+                    Contact Us
+                  </Button>
```

Comment:
Could you consider overriding the built-in primary color of material-UI since on the designs we are having `2196F3`. Just to be consistent. Anyways you did good work.

Reply:
thanks @SimonImanigirimpuhwe, I'm working on it soon.

Reply:
@SimonImanigirimpuhwe you can kindly review again I responded to your feedback.

Reply:
Looks nice!